src/test/regress/bin/normalize.sed

```diff
@@ -297,3 +297,11 @@ s/"Total Cost": [0-9].[0-9]+/"Total Cost": xxxx/g
 
 # normalize gpids
 s/(NOTICE:  issuing SET LOCAL application_name TO 'citus_rebalancer gpid=)[0-9]+/\1xxxxx/g
+
+# PG13 changes bgworker sigterm message, we can drop that line with PG13 drop
+s/(FATAL: terminating).*Citus Background Task Queue Executor.*(due to administrator command)\+/\1 connection \2                    \+/g
```

Comment:
This line seems to be part of line 120 of this same file. Was this some incorrect merge conflict resolution?

Reply:
hmm, yes, fixed now. Thanks